core/src/main/scala/kafka/server/ConfigHelper.scala

```diff
@@ -80,7 +80,7 @@ class ConfigHelper(metadataCache: MetadataCache, config: KafkaConfig, configRepo
   def describeConfigs(resourceToConfigNames: List[DescribeConfigsResource],
                       includeSynonyms: Boolean,
                       includeDocumentation: Boolean): List[DescribeConfigsResponseData.DescribeConfigsResult] = {
-    resourceToConfigNames.map { case resource =>
```

Comment:
Previous code was the standard in really old version of Scala, since then anonymous function style is preferred.